gentle_gnomes/src/indicator.py

```diff
@@ -50,7 +50,7 @@ def _calc_slope(x: np.ndarray, y: np.ndarray) -> float:
         return slope
 
 
-def get_top_indicators(city: City, n: int = 5) -> Tuple[Indicator, ...]:
+async def get_top_indicators(city: City, n: int = 5) -> Tuple[Indicator, ...]:
```

Comment:
No need to make something async if it's not going to be awaiting anything. But based on the previous review you'll probably have to await  `_populate_data` in here instead.